dll/ntdll/ldr/ldrpe.c

```diff
@@ -703,7 +703,7 @@ LdrpWalkImportDescriptor(IN LPWSTR DllPath OPTIONAL,
                 DPFLTR_WARNING_LEVEL,
                 "LDR: LdrpWalkImportDescriptor() failed to probe %wZ for its "
                 "manifest, ntstatus = 0x%08lx\n",
-                &LdrEntry->FullDllName, Status);
+                &LdrEntry->FullDllName, Status2);
```

Comment:
So `//Status = Status2; // FIXME: Ignore that error for now` at the top can be removed?

Reply:
No: although that commit was the "unhider", the FIXME itself is a separate matter.